recipes/wildmagic/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "WildMagic" %}
+{% set version = "5.14" %}
+{% set version_major = "5" %}
+{% set version_minor = "14" %}
+{% set sha256 = "951330bca2328d537dac2459bf154845295b8c4576a0adca5ffbf31b5a778d80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}{{ version_major }}p{{ version_minor }}.zip
+  url: http://www.geometrictools.com/Downloads/WildMagic{{ version_major }}p{{ version_minor }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+    - wm5_path.patch
```

Comment:
wildmagic is superseded by GTEngine, and there hasn't been a release since last year. I somehow doubt they will want to make changes to the build system at this point. But I can try if you like.
This patch is similar to what debian does (they don't have to make it relocatable, so theirs is simpler.)